src/translations/en.js

```diff
@@ -0,0 +1,39 @@
+import { minSymbol } from "../components/FirstRegistrationForm"
+import { maxSymbol } from "../components/FirstRegistrationForm"
+
+export const messages_en = {
+"app.name": "Lessoner",
+"app.studio": "My Studio",
+"app.categories": "Categories",
+"app.lessons": "Lessons",
+"app.about": "About",
+"app.login": "LogIn",
+"app.registration": "Register",
+"app.header.login": "Log in",
+"app.header.myStudio": "My studio",
+"app.header.goStudy":"Go study",
+"app.footer": "Footer",
+"app.lessoner": "Lessoner",
+"app.sign_in": "Sign_in",
+"app.sign_up": "Sign_up",
+"app.birthdaylabel": "When is your birthday?",
+"app.checkbox": "I agree to the processing of my personal data in accordance with the",
+"app.checkbox.terms": "Terms",
+"app.email.name": "Email",
+"app.firstRegistrationForm.existsInDb": "This email address is already registered",
+"app.firstRegistrationForm.invalidationRules": "Please enter a valid email address",
+"app.firstRegistrationForm.passwordRegEx": `An invalid character is present in the password. Password must be between ${minSymbol} and ${maxSymbol} characters; upper or lower case Latin letters (a–z, A–Z); numbers from 0 to 9; symbols ! # $ % & ' * + - / = ? ^ _  { | } ~`,
```

Comment:
Use templete here without import of constants. **Example:**

"app.form.message": 'Your password must be {minValue} - {maxValue}'

then when using:

intl.formatMessage({ id: 'app.form.message', {minValue: 1, maxValue: 5 }});

https://formatjs.io/docs/intl/#message-formatting-fallbacks

Reply:
+

Reply:
agreed